statsmodels/robust/norms.py

```diff
@@ -145,11 +157,17 @@ def psi(self, z):
         return np.asarray(z)
 
     def weights(self, z):
-        """
+        r"""
         The least squares estimator weighting function for the IRLS algorithm.
 
         The psi function scaled by the input z
 
+        .. math::
+
+            \text{weights}: \mathbb{R}^n \to \mathbb{R}^n
+
+            \text{weights}(z) = \frac{\psi(z)}{z}
```

Comment:
This one doesn't seem to match the answer. Looks like weights(z) = 1

Reply:
@bashtage  Thanks for the feedback. I see I need to go over the entire file more thoroughly.

I would have a quick question:

Would it be an overkill to write the following
$$\text{weights}(z) = \text{weights}(z) = \mathbf{1}_n, \quad \text{where }\mathbf{1}_n = (1,1,...,1)^t \in \mathbb{R}^n.$$

 whenever there is a vector of ones returned? 

Or would $\text{weights}(z) = \text{weights}(z) = \mathbf{1}_n$ without any explanation suffice?